src/sortStudents.ts

```diff
@@ -1,16 +1,61 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+type SortOrder = 'asc' | 'desc';
 
+export type { SortOrder };
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+function getAverageGrade(grades: number[]): number {
+  return grades.reduce((acc, value) => (
+    acc + value), 0) / grades.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const sortedObject = [...students];
```

Comment:
Use more descriptive names and don't use object types in their names
```suggestion
  const copyOfStudents = [...students];
```